src/components/description/DescriptionSubscriber.js

```diff
@@ -34,45 +35,34 @@ export default function DescriptionSubscriber(props) {
   const [{
     dataset,
     spatialImageLayer: rasterLayers,
-  }] = useCoordination(COMPONENT_COORDINATION_TYPES.description, coordinationScopes);
-
-  const [
-    isReady,
-    setItemIsReady,
-    setItemIsNotReady, // eslint-disable-line no-unused-vars
-    resetReadyItems,
-  ] = useReady(
-    DESCRIPTION_DATA_TYPES,
-  );
-
-  // Reset loader progress when the dataset has changed.
-  useEffect(() => {
-    resetReadyItems();
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [loaders, dataset]);
+  }] = useCoordination(COMPONENT_COORDINATION_TYPES[ViewType.DESCRIPTION], coordinationScopes);
 
   // Get data from loaders using the data hooks.
   const [description] = useDescription(loaders, dataset);
-  const [raster, imageLayerLoaders, imageLayerMeta] = useRasterData(
-    loaders, dataset, setItemIsReady, () => {}, false,
+  const [{ image }, imageStatus] = useImageData(
+    loaders, dataset, addUrl, false, {}, {},
+    {}, // TODO: which properties to match on
```

Comment:
This is a good question....

Reply:
I think https://github.com/vitessce/vitessce/issues/830 should be addressed first.